build/charts/theia/provisioning/dashboards/node_to_node_dashboard.json

```diff
@@ -168,9 +168,7 @@
       "interval": "60s",
       "options": {
         "legend": {
-          "calcs": [
-            "mean"
-          ],
+          "calcs": [],
```

Comment:
As we removed the "mean" metric from legend, we might need to regenerate new screenshots and replace those in the network-flowvisibility.md documentation. Could you send me the related new screenshots? I'll upload them to S3 bucket and give you back the links to put in the doc.

Reply:
Agree, could you also attach some screenshots related to the UI changes in this PR? This will be very helpful for code review.

Reply:
I've updated these screenshots in the description above. Please take a look to see if these photos are acceptable. Thanks.